code/src/App.css

```diff
@@ -0,0 +1,449 @@
+body::after {
+  content: ""; 
+  background: url("/assets/icecream.jpg")center center;
+  opacity: 0.5;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0; 
+  position: fixed; 
+  z-index: -1;
+  background-size:cover;
+  height: 100vh;
```

Comment:
This is interesting! I guess this is a part of your solution to show the last summary? What is it ::after really? I have mostly seen this on links where the after is then after click for example. Would be nice to add an explanatory comment?